src/screens/OtherProfile/index.js

```diff
@@ -706,84 +715,84 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
     alignItems: 'center',
     justifyContent: 'space-between',
-    backgroundColor: 'white'
+    backgroundColor: COLORS.white
   },
   wrapButton: {
     flex: 1,
     flexDirection: 'row',
     alignItems: 'center',
     justifyContent: 'flex-end'
   },
-  buttonFollowing: {
+  buttonFollowing: (isAnon) => ({
     width: 88,
     height: 36,
     flexDirection: 'row',
     justifyContent: 'center',
     alignItems: 'center',
     borderWidth: 1,
-    borderColor: colors.bondi_blue,
+    borderColor: isAnon ? COLORS.anon_primary : COLORS.signed_primary,
     borderRadius: 8
-  },
-  buttonFollow: {
+  }),
+  buttonFollow: (isAnon) => ({
     width: 88,
     height: 36,
     flexDirection: 'row',
     justifyContent: 'center',
     alignItems: 'center',
     borderRadius: 8,
-    backgroundColor: colors.bondi_blue,
-    color: colors.white
-  },
-  textButtonFollowing: {
+    backgroundColor: isAnon ? COLORS.anon_primary : COLORS.signed_primary,
+    color: COLORS.white
+  }),
+  textButtonFollowing: (isAnon) => ({
     fontFamily: fonts.inter[600],
     fontWeight: 'bold',
     fontSize: 12,
-    color: colors.bondi_blue
-  },
+    color: isAnon ? COLORS.anon_primary : COLORS.signed_primary
+  }),
   textButtonFollow: {
     fontFamily: fonts.inter[600],
     fontWeight: 'bold',
     fontSize: 12,
-    color: colors.white
+    color: COLORS.white
   },
   btnBottom: {
     position: 'absolute',
     width: dimen.size.PROFILE_ACTION_BUTTON_RADIUS,
     height: dimen.size.PROFILE_ACTION_BUTTON_RADIUS,
     right: 20,
     bottom: dimen.size.FEED_ACTION_BUTTON_HEIGHT_FROM_BOTTOM,
-    backgroundColor: colors.darkBlue,
+    backgroundColor: COLORS.signed_primary,
     borderRadius: 30,
     flexDirection: 'row',
     justifyContent: 'center',
     alignItems: 'center'
   },
   tabsFixed: {
     width,
-    borderBottomColor: colors.alto,
+    borderBottomColor: COLORS.lightgrey,
     borderBottomWidth: 1,
     paddingLeft: 20,
     paddingRight: 20,
     flexDirection: 'row',
     position: 'absolute',
     top: 0,
     zIndex: 2000,
-    backgroundColor: colors.white
+    backgroundColor: COLORS.white
   },
   containerFlatFeed: {
     // padding: 20,
     flex: 1
   },
-  btnMsg: {
+  btnMsg: (isAnon) => ({
     width: 36,
     height: 36,
     borderWidth: 1,
     borderRadius: 8,
-    borderColor: colors.bondi_blue,
+    borderColor: isAnon ? COLORS.anon_primary : COLORS.signed_primary,
     flexDirection: 'row',
     justifyContent: 'center',
     alignItems: 'center'
-  },
+  }),
   containerLoading: {
     height: '100%',
     width: '100%',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [681-795]

The changes in this hunk seem to be related to the introduction of a new color scheme based on user anonymity. The logic seems fine, but it's important to ensure that these color changes do not negatively affect the accessibility of the application. For example, make sure there is enough contrast between text and background colors for readability.



<!-- This is an auto-generated comment by OSS CodeRabbit -->